features/steps/steps.py

```diff
@@ -36,11 +36,38 @@ def when_i_run_command(context, command, user_spec):
     context.process = process
 
 
+@when('I attach "{token}" {user_spec}')
+def when_i_attach_token(context, token, user_spec):
+    token = context.config.contract_token
+    if not token:
+        context.scenario.skip(
+            "Skipping test because you didn't provide a token"
+        )
+    when_i_run_command(context, "ua attach %s" % token, user_spec)
+
+
 @then("I will see the following on stdout")
 def then_i_will_see_on_stdout(context):
     assert_that(context.process.stdout.strip(), equal_to(context.text))
 
 
+@then("stdout will include")
+def then_and_stdout_will_include(context):
+    assert_that(context.process.stdout.strip(), contains_string(context.text))
+
```

Comment:
I think you separately need a new @then step as commented above

```suggestion
+@then("stdout matches regexp")
+def then_stdout_matches_regexp(context):
+    assert_that(context.process.stdout.strip(), matches_regexp(context.text))
```